src/Symfony/Component/Serializer/Tests/Normalizer/GetSetMethodNormalizerTest.php

```diff
@@ -323,7 +390,7 @@ protected function getNormalizerForMaxDepth(): NormalizerInterface
     protected function getDenormalizerForObjectToPopulate(): DenormalizerInterface
     {
         $classMetadataFactory = new ClassMetadataFactory(new AnnotationLoader(new AnnotationReader()));
-        $normalizer = new GetSetMethodNormalizer($classMetadataFactory, null, new PhpDocExtractor());
+        $normalizer = new GetSetMethodNormalizer($classMetadataFactory, [], new PhpDocExtractor());
```

Comment:
null should still be accepted, no need to deprecate it.

Reply:
Why would you want to keep passing null? What would it mean?

Reply:
unnecessary deprecation should be avoided, that's my point.

Reply:
OK :+1: